### DISCUSSION TITLE
how can I be a member of the pdf organization/community?

### DISCUSSION THREAD

**Discussion by ghost** [pdf-association/pdf-issues#249] [Q&A] [answered]
Hi everyone.

I would like to know what are the steps, processes or way to be a member of the pdf community. Or how I can contribute to the pdf community. A curious question: - *How can I be a member of the pdf organization/community?*

#### other supplementary questions could be
- What are the technical skills required? 
- Or what are the skills required? 
- or what are the required soft skills?
- Do I have to pay for anything to be a member of the pdf community?
- Is there an age restriction to join the pdf community?
- Do I need to have an invitation from a member of the pdf community to be accepted?

<br/>

*If someone replies to this message, I would be happy for the feedback.*

**Comment by petervwyatt** [comment, marked as answer]
Becoming part of the PDF community is easy - you already are by asking questions here 😁.

The 2 primary ways of getting more actively involved are via the PDF Association (industry body) or your national standards body to join _only_ the ISO discussions. The PDF Association has an official liaison with ISO and a mirror IP policy so joining the PDF Association will also give you access to all ISO standardization activities. In addition, the PDF Association has various working groups working on other PDF topics that are not part of ISO - see https://www.pdfa.org/community/.

Contact your national standards body to find out how to join ISO PDF standardization activities in ISO TC 171 SC 2. Your country may already be a Participating (P) or Observing (O) member of this ISO Technical Committee - see https://www.iso.org/committee/53674.html for the list of P and O countries. Each national standards body have their own requirements about how to join and whether there is any cost involved for individuals.

You do NOT need PDF technical skills to participate - in fact, the PDF community is always looking for people from industries that are "end users" of PDF such as archivists, preservationists, engineering/3D, graphic arts, accessibility, AT vendors, remediators, LaTeX experts, etc. (just to name a few).

See [this page](https://www.pdfa.org/become-a-member/) on how to join the PDF Association - note that there are options for non-profit organizations to have a liaison membership and government and academic organizations to have institutional membership depending on who you are associated with. The majority of resources published by the PDF Association are free to access - here at GitHub, YouTube channel, the website, etc.

**Reply by ghost** [reply]
Hi petervwyatt, thank you for feedback.
